Makefile

```diff
@@ -271,13 +271,24 @@ go_imports: check_go_version ## Run goimports on all go files
 ### Tests ###
 #############
 
-.PHONY: test_e2e
-test_e2e: acc_initialize_pubkeys_warn_message ## Run all E2E tests
+.PHONY: acc_initialize_pubkeys_warn_message test_e2e_env
+test_e2e_env: ## Setup the default env vars for E2E tests
 	export POCKET_NODE=$(POCKET_NODE) && \
 	export APPGATE_SERVER=$(APPGATE_SERVER) && \
-	POKTROLLD_HOME=../../$(POKTROLLD_HOME) && \
+	export POKTROLLD_HOME=../../$(POKTROLLD_HOME)
```

Comment:
```suggestion
	export POKTROLLD_HOME=../../$(POKTROLLD_HOME) # because E2E tests live under e2e/tests
```

Reply:
After thinking about it, this is extra unnecessary information. Resolving and ignoring.